src/http_response.cpp

```diff
@@ -54,12 +54,22 @@ void http_response::shoutCAST() {
     response_code |= http::http_utils::shoutcast_response;
 }
 
+namespace {
+static inline http::header_view_map to_view_map(const http::header_map & hdr_map) {
+    http::header_view_map view_map;
+    for (const auto & item : hdr_map) {
```

Comment:
nit: keep '&' attached on the left

Reply:
not sure you did this?

Reply:
I noticed others below - can you make sure you keep the reference '&' consistently together (left attached) to the type in all of your changes?

I am aware the code is inconsistent (even in some of the rows you are modifying), but let's make sure we fix everything we are touching or creating anew. I need to go and have a pass at cleaning everything up later for the rest..

Reply:
I think I got them all now. I would suggest configuring something clang-format, though the initial application of it will likely make quite a few changes that are formatting-only.

Reply:
Yep, there is a change pending on that (with a few thousands fixes) that I am going through.